### ISSUE TITLE
Allow disabling shortcuts

### ISSUE THREAD

**Issue by jonocodes** [odebroqueville/contextSearch#153]
There is no way to blank them out in the settings.

**Comment by vbmark** [comment]
Yes, this ruins the extension.  Can't use it with shortcuts enabled.

**Comment by odebroqueville** [comment]
@jonocodes Sorry for my belated response. I've been pretty busy! I'm not able to reproduce the issue that you're having. I've recorded an [example](https://imgur.com/a/5iKihEe) where I place the cursor in the text field at the end of the keyboard shortcut for Adobe. I then press the Backspace key until the text fiels is empty. Finally, I click anywhere outside the text field. I then refreshed the Options page twice to make sure that the changes had been saved. I encountered no issues! Please provide more details, i.e. the steps to help reproduce the bug you're encountering. Are you on Mac, Linux or Windows? Which version of Context Search are you using?

@vbmark How does having shortcuts ruin your experience? 

In general, you should be able to select any keyboard shortcut, then press the Delete or Backspace key and click outside the text field to remove the keyboard shortcut. I haven't had any problems with this on macOS, but I'm not able to test for Windows or Linux.

**Comment by jonocodes** [comment]
I appear to be using 4.5.5.
Now trying this on my mac, it works as you show in the video.
But the problem happens in Linux. If I delete the shortcut, it replaces it with 'backspace' instead of blanking it out. 
And it I 'cut' it, then refreshing the page brings back the old shortcut.
Thanks for taking a look.

**Comment by vbmark** [comment]
Hi @odebroqueville 

Thank you for your response.  Like the original poster, I could not remove the shortcuts so in the browser when I tried to do ctrl+c for example to copy something, the Encyclopaedia Britannica page would open.  So I had to disable the extension.

However, I just tried deleted the shortcuts again and this time they are blanking out just fine and staying blanked out.

I don't know what changed to cause it to work properly now but if I happen to figure it out I'll share it here.

Thank you!
